content/en/docs/distributions/gke/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Kubeflow on Google Cloud"
-description = "Running Kubeflow on Kubernetes Engine and Google Cloud Platform"
-weight = 20
+title = "Kubeflow Blueprint On Google Cloud"
+description = "Documentation for Kubeflow Blueprint On Google Cloud"
```

Comment:
I think `Kubeflow on Google Cloud` is more straightforward, it seems not clear to me why we are changing it.

Reply:
@zijianjoy It's for the same reasons as we are renaming the other distributions. So that we (the Kubeflow community) aren't seen to "endorse" specific distributions for specific platforms. This is because `kubeflow.org` is a "neutral" place, but obviously, Google can use other channels to suggest their own distribution!

__NOTE:__ I only chose "Kubeflow Blueprint On Google Cloud" because that was the name on your [kubeflow/gcp-blueprints](https://github.com/kubeflow/gcp-blueprints) repo, so I assumed it was the "name" of your distribution. If there is a better name, please tell me and I will update it.

Reply:
Can we use this name instead? `Google Cloud distribution for Kubeflow`. 

Blueprint is a deprecated term in kpt which describes a package or a bundle, kpt doesn't use this term any more.

Reply:
@zijianjoy If that's the name you want to use, it sounds fine, but I think you should also update the title in your repo README to prevent confusion.

https://github.com/kubeflow/gcp-blueprints

Reply:
@zijianjoy I have rebased and updated the name, but to `Google Cloud distribution of Kubeflow`.

NOTE: I used "of" rather than "for" (as I think this is more grammatically correct), tell me if you agree?

Reply:
Sounds good @thesuperzapper , I have also updated the repo name to `Google Cloud distribution of Kubeflow` so both documentation and repo are aligned.